plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/EverytradeCSVParserValidator.java

```diff
@@ -0,0 +1,51 @@
+package io.everytrade.server.plugin.impl.everytrade.parser;
+
+import io.everytrade.server.plugin.api.parser.TransactionCluster;
+import io.everytrade.server.plugin.impl.everytrade.parser.exchange.ExchangeBean;
+import io.everytrade.server.util.CurrencyUtil;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class EverytradeCSVParserValidator {
+
+    public static Map<String,String> parsedSymbol(String value) {
```

Comment:
I dont like the fact that this method return map of base and quote and it is parsed on mulitple places in same way. You can return CurrencyPair instance here and it will be much simpler, no Map etc..